src/templates/next/components/shared/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import { BiLeftArrowAlt, BiRightArrowAlt } from "react-icons/bi"
+import type { PaginationProps } from "../../../types/Pagination"
+
+export const Pagination = ({
+  totalItems,
+  itemsPerPage,
+  currPage,
+  setCurrPage,
+}: Omit<PaginationProps, "type">) => {
+  const totalPages = Math.ceil(totalItems / itemsPerPage)
+
+  return (
+    <nav className="flex flex-row w-full" aria-label="Pagination">
+      {/* Previous button */}
+      <button
+        className="flex flex-row p-1 gap-1 align-middle cursor-pointer disabled:cursor-not-allowed disabled:text-interaction-sub disabled:hover:bg-transparent hover:bg-interaction-main-subtle-hover"
+        aria-label="Previous page"
+        disabled={currPage === 1}
+        onClick={() => {
+          if (currPage !== 1) {
+            setCurrPage(currPage - 1)
+          }
+        }}
+      >
+        <BiLeftArrowAlt className="text-2xl my-auto" />
+        <p className="hidden xs:inline underline text-lg">Previous</p>
```

Comment:
i assume this is for a11y purposes but why not jjust attach a `label` to the button

Reply:
The label is added at the top in the button using `aria-label`.